documentation/course/e2/README.md

```diff
@@ -852,50 +880,50 @@ TODO @piterek130: uzupełnić tabelę
   </tr>
   <tr>
     <th>Nazwa aplikacji</th>
-    <td>Aplikacja źródłowa (aplikacja, która inicjuje integrację na poziomie logicznym, czyli w większości przypadków aplikacja będąca przy końcu strzałki bez grotu)</td>
-    <td>Aplikacja docelowa (aplikacja przy końcu strzałki z grotem)</td>
+    <td>JakPrzyjade</td>
+    <td>Tpay</td>
   </tr>
   <tr>
     <th>Technika integracji</th>
-    <td>FTP/SSH/JDBC/ODBC/Oracle DBLink/SAP JCo/SAP RFC/ SOAP/HTTP / SOAP/HTTPS /Własny/...</td>
-    <td>FTP/SSH/JDBC/ODBC/Oracle DBLink/SAP JCo/SAP RFC/ SOAP/HTTP / SOAP/HTTPS /Własny/...</td>
+    <td>REST API / HTTPS</td>
+    <td>REST API / HTTPS</td>
   </tr>
   <tr>
     <th>Mechanizm autentykacji</th>
-    <td>Brak - niezalecane/HTTP Basic/WS-Security/Kerberos/Oracle Username and Password/...</td>
-    <td>Brak - niezalecane/HTTP Basic/WS-Security/Kerberos/Oracle Username and Password/...</td>
+    <td>OAuth 2.0</td>
+    <td>OAuth 2.0</td>
   </tr>
   <tr>
     <th>Kontrakt danych</th>
-    <td colspan="2">Lista wymienianych obiektów biznesowych (np. Partner Handlowy, Konto Umowy itp.)</td>
+    <td colspan="2">OAuth 2.0 Access Token, kwota, dane pasażera, opis transakcji, status transakcji</td>
   </tr>
   <tr>
     <th>Czy interfejs manipuluje na danych wrażliwych (RODO)?</th>
-    <td colspan="2">Tak/Nie + jakie dane objęte RODO</td>
+    <td colspan="2">Tak, przetwarzane są dane osobowe płatników, takie jak adres e-mail, imię i nazwisko oraz dane karty kredytowej</td>
   </tr>
   <tr>
     <th>Wykorzystywane oprogramowanie pośredniczące (middleware)</th>
-    <td colspan="2">Brak/SAP PI/ESB/ODS/ESB i ODS/...</td>
+    <td colspan="2">Brak</td>
   </tr>
   <tr>
     <th>Strona inicjująca</th>
-    <td colspan="2">Aplikacja inicjująca połączenie na poziomie technicznym</td>
+    <td colspan="2">JakPrzyjade</td>
   </tr>
   <tr>
     <th>Model komunikacji</th>
-    <td colspan="2">Synchroniczny na żądanie użytkownika/Asynchroniczny sterowany harmonogramem/Asynchroniczny wyzwalany zdarzeniem/...</td>
+    <td colspan="2">Synchroniczny na żądanie użytkownika (tworzenie transakcji), asynchroniczny wyzwalany zdarzeniem (powiadomienie o statusie płatności).</td>
   </tr>
   <tr>
     <th>Wydajność</th>
-    <td colspan="2">Jak często jest wywoływany interfejs, należy przedstawić największe wymaganie wydajnościowe w jednostce czasu dla której wymaganie musi być spełnione, np. 1000 / godz. Im krótsza jednostka czasu tym wymaganie ściślejsze.</td>
+    <td colspan="2">Szacowana liczba wywołań: 16 000 / godz</td>
```

Comment:
Please include the calculations which led to this estimation above or below the table.